test/testdata/config-example.yaml

```
@@ -3,39 +3,37 @@ TotalRecords: 5031
 BatchSize: 700
 Columns:
   user_id:
-    type: int/uniform
-    minVal: 1
-    maxVal: 2000
+    Type: int/uniform
+    MinVal: 1
+    MaxVal: 2000
   txid:
-    type: string/uuid
+    Type: string/uuid
   btc:
-    type: float
-    minVal: 0.0
-    maxVal: 123.4
+    Type: float
+    MinVal: 0.0
+    MaxVal: 123.4
   datetime:
-    type: datetime/rand
-    minVal: 2012-01-01 00:00:00
+    Type: datetime
+    MinVal: 2012-01-01 00:00:00
```

Add max val to make the example deterministic for tests.
```suggestion
    MinVal: 2012-01-01 00:00:00
    MaxVal: 2022-01-01 00:00:00
```